### PR TITLE
Telmos/acag int jumpbox

### PR DESCRIPTION
**Pull request by telmosampaio**

### CODE REVIEW

**Comment by bennage** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md](https://github.com/mspnp/architecture-center/blob/telmos/acag-int-jumpbox/docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/cloud-adoption-guide/intermediate-stage/jumpbox-explainer?branch=pr-en-us-444) | [Details](#user-content-docs%2Fcloud-adoption-guide%2Fintermediate-stage%2Fjumpbox-explainer.md)

<a name="docs%2Fcloud-adoption-guide%2Fintermediate-stage%2Fjumpbox-explainer.md"></a>
### [docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md](https://github.com/mspnp/architecture-center/blob/telmos/acag-int-jumpbox/docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md)
 - **Line 22**: **[Warning]** Invalid file link:(~/reference-architectures/virtual-machines-linux/jumpbox.md).
 - **Line 23**: **[Warning]** Invalid file link:(~/reference-architectures/virtual-machines-linux/linux.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C5%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201803052248417906-444%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LXuRcw%2B9J4EElKBqRmU8yIoNpnhJ25sHyXz7884jICo%3D&st=2018-03-05T22%3A44%3A40Z&se=2018-04-05T22%3A49%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: "Explainer: what is a jumpbox or bastion host?"
+description: Defines the terms 'jumpbox' and 'bastion host'
+author: telmosampaio
+---
+
+# Explainer: what is a jumpbox?
+
+In the foundational adoption stage, you learned about using a single infrastructure-as-a-service (IaaS) virtual machine (VM). Most IaaS workloads will be composed by more than a single VM. They can encompass several VMs running different services, such as a web UI, distributed micro services, and database servers. 
+
+Different users need to access these separate VMs in a secure fashion, without you necessarly providing direct access to the VMs themselves. You can do that by using a **jumpbox**, **bastion host**, or **secure administrative host**.
```
Comment (Petertaylor9999) [inline docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md]: Last sentence: "You can do that by using a jumpbox, also known as a bastion host or secure administration host. Then you can do away with the note below.

File: docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# Explainer: what is a jumpbox?
+
+In the foundational adoption stage, you learned about using a single infrastructure-as-a-service (IaaS) virtual machine (VM). Most IaaS workloads will be composed by more than a single VM. They can encompass several VMs running different services, such as a web UI, distributed micro services, and database servers. 
+
+Different users need to access these separate VMs in a secure fashion, without you necessarly providing direct access to the VMs themselves. You can do that by using a **jumpbox**, **bastion host**, or **secure administrative host**. 
+
+> [!NOTE]
+> The terms **jumpbox**, **bastion host**, and **secure administrative host** are used interchangebly in the industry. We will use the term **jumpbox** in this document to facilitate reading.
```
Comment (Petertaylor9999) [inline docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md]: See above.

File: docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+A **jumpbox** is a computer used to manage devices in different security zones. In public clouds, jumpboxes become extremelly valuable as a way to manage VMs that are protected from access from the public Internet. The jumpbox VM itself might be exposed to the Internet, and have access to VMs in your workload that are not exposed to the public. That creates an isolation boundary, or **DMZ**, that you control to provide management access to secure resources.
```
Comment (Petertaylor9999) [inline docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md]: "In public clouds, a jumpbox is valuable as a way to manage VMs..."

You may also consider something like:

"The jumpbox sits between the unprotected network and the private secure network. Access to the VMs in the private secure network is limited to that which the jumpbox provides. This narrows the potential attack surface to only that which the jumpbox allows, creating an isolation boundary...."

File: docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Explainer: what is a jumpbox or bastion host?"
+description: Defines the terms 'jumpbox' and 'bastion host'
+author: telmosampaio
+---
+
+# Explainer: what is a jumpbox?
+
+In the foundational adoption stage, you learned about using a single infrastructure-as-a-service (IaaS) virtual machine (VM). Most IaaS workloads will be composed by more than a single VM. They can encompass several VMs running different services, such as a web UI, distributed micro services, and database servers. 
+
```
Comment (Petertaylor9999) [inline docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md]: I think you should add a paragraph that defines the problem space that a jumpbox solves here. The problem is that we need to SSH or RDP to secure VMs, but we don't want to put public IP addresses on them. So, we need to have a public IP address that is not connected to a VM on the protected private network.

**Comment by bennage** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md](https://github.com/mspnp/architecture-center/blob/telmos/acag-int-jumpbox/docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/cloud-adoption-guide/intermediate-stage/jumpbox-explainer?branch=pr-en-us-444) | [Details](#user-content-docs%2Fcloud-adoption-guide%2Fintermediate-stage%2Fjumpbox-explainer.md)

<a name="docs%2Fcloud-adoption-guide%2Fintermediate-stage%2Fjumpbox-explainer.md"></a>
### [docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md](https://github.com/mspnp/architecture-center/blob/telmos/acag-int-jumpbox/docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md)
 - **Line 19**: **[Warning]** Invalid file link:(~/reference-architectures/virtual-machines-linux/jumpbox.md).
 - **Line 20**: **[Warning]** Invalid file link:(~/reference-architectures/virtual-machines-linux/linux.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C8%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201803081725023846-444%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tLRE93EmBEeZd85epzW%2B4AnJ7xgRLyXd7JZ8zoo3l2o%3D&st=2018-03-08T17%3A21%3A31Z&se=2018-04-08T17%3A26%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request closed without merging**